examples/with-universal-configuration-runtime-docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:9-slim AS builder
```

Comment:
Lets use `mhart/alpine-node`

Reply:
Sure, the `mhart/alpine-node` image is smaller than official one.